modules/administration-guide/partials/con_authentication-modes.adoc

```diff
@@ -11,9 +11,13 @@ Single-user mode requires no authentication and anyone can access all cluster re
 
 * H2 database is used.
 
-{prod-short} deployed in single-user mode has no authentication and anyone who can access the URL of the {prod-short} deployment sees all workspaces and owns all resources. Because the deployment in this mode requires fewer containers, RAM and CPU requirements are lower. This mode is useful when the whole {prod-short} deployment is used by a single person or for lowering resources used. As the server does not authenticate, actions of multiple users logged in the same workspace can interfere with each other.
+{prod-short} deployed in single-user mode has no authentication and anyone who can access the URL of the {prod-short} deployment sees all workspaces and owns all resources. The fact that this mode has no authentication leads to it deprecation and removal. It's highly recommended migrating to multiuser mode.
```

Comment:
```suggestion
```

We don't need to document the previous behaviour in the docs. Therefor we can safedly drop the complete paragraph.